Makefile

```diff
@@ -6,6 +6,9 @@ all: install
 fmt:
 	go fmt ./...
 
+fmt-strict:
+	@exit $(shell go fmt ./... | wc -l)
+
```

Comment:
can this be moved to the .travis file? I don't feel like it belongs in the Makefile.